pyproject.toml

```diff
@@ -49,6 +49,7 @@ dependencies = [
     "safetensors",
     "torch",
     "transformers",
+    "typer-slim>=0.12.0",
```

Comment:
Double check on entrypoints are added and working -- "speculators" -> CLI file

Reply:
https://github.com/neuralmagic/guidellm/blob/main/pyproject.toml#L111:

[project.entry-points.console_scripts]
guidellm = "guidellm.\_\_main__:cli"